app/colors/compact.css

```diff
@@ -0,0 +1,18 @@
+/*
+ * GLOBAL
+*/
+#app {flex-direction: row;flex-wrap: wrap;}
+
+/*
+ * APP
+*/
+/* navbar & tabs */
+#navbar {width: 40vw; height: 1.8em;}
+#logo {height: 1.2em;width: 1.2em;min-width: 1.2em;min-height: 1.2em; padding:0.2em;}
+#mode-container {width: 6em;}
+#mode {font-size: 1.1em;}
+
```

Comment:
I don't really think this creates extra space, it only changes the sizes slightly, but the margins remain. Removing lines 11-13 will make the theme simpler, and easier to understand, while at the same time maintaining the default look of Vieb better. What do you think?

Reply:
Perfect I just did a couple of changes to keep the default look, it is still good for me and happy if we can share it.

Reply:
Thanks, looks good now!